datadog/__init__.py

```diff
@@ -117,6 +118,12 @@ def initialize(
 
     :param hostname_from_config: Set the hostname from the Datadog agent config (agent 5). Will be deprecated
     :type hostname_from_config: boolean
+
+    :param cardinality: Set the global cardinality for all metrics. \
+        Possible values are "none", "low", "orchestrator" and "high".
+        Can also be set via the DATADOG_CARDINALITY or DD_CARDINALITY environment variables.
```

Comment:
Should we link to possible values to the [tag cardinality section](https://docs.datadoghq.com/getting_started/tagging/assigning_tags/?tab=containerizedenvironments#environment-variables) in our docs with something like to "to see all possible values check our docs"

Reply:
Discussed together as it would be better to also include documentation in case of value changes in the future.

Reply:
Sadly adding the link would break the linter.